src/PhpSpreadsheet/Spreadsheet.php

```diff
@@ -790,8 +791,8 @@ public function setActiveSheetIndexByName(string $worksheetName): Worksheet
     public function getSheetNames(): array
     {
         $returnValue = [];
-        $worksheetCount = $this->getSheetCount();
-        for ($i = 0; $i < $worksheetCount; ++$i) {
+
+        for ($i = 0; $i < $this->getSheetCount(); ++$i) {
```

Comment:
Once again, you are moving a function call from outside the loop to inside. This part of the change will not be approved.

Reply:
Reverted in latest commit.